drivers/spi/Kconfig.mcux_dspi

```diff
@@ -1,11 +1,43 @@
 # MCUXpresso SDK SPI
 
 # Copyright (c) 2016, Freescale Semiconductor, Inc.
-# Copyright (c) 2017, NXP
+# Copyright (c) 2017, 2021, NXP
 # SPDX-License-Identifier: Apache-2.0
 
 config SPI_MCUX_DSPI
 	bool "MCUX SPI driver"
 	depends on HAS_MCUX && CLOCK_CONTROL
 	help
 	  Enable support for mcux spi driver.
+
+if SPI_MCUX_DSPI
+
+config DSPI_MCUX_EDMA
+	bool "ENABLE EDMA for DSPI driver"
+	depends on HAS_MCUX && HAS_MCUX_EDMA
+	help
+	  Enable the MCUX DSPI driver.
+
+config DSPI_INIT_PRIORITY
+	int "Init priority"
+	default 70
+	help
+	  Device driver initialization priority.
```

Comment:
Remove this and use `SPI_INIT_PRIORITY` from `drivers/spi/Kconfig` instead

Reply:
ok